fnf/README.md

```diff
@@ -0,0 +1,16 @@
+## How to setup and use a PostgreSQL DB ##
+Install PostgreSQL:
+* For macOS users, the fastest way is to download the [Postgres.app](https://postgresapp.com/) and follow the installation instructions. To connect to a database, make sure that the app is running.
+* For all other users, you should be able to find a suitable distribution [here](https://www.postgresql.org/download/).
+
+Then, run `python mag_orm.py` to create the project's database (`disinfo`) and its tables.
+
+Note that the `.env` file contains two connections to PostgreSQL in the following format:
+
+``` python
+postgresdb = postgres+psycopg2://NAME:PASSWORD@localhost:5432/disinfo
+test_postgresdb = postgres+psycopg2://NAME:PASSWORD@localhost:5432/postgres
```

Comment:
Note: `postgres+psycopg2://postgres@localhost/DB_NAME` worked for me on Linux. 
Wondering if this works on macOS as it is simpler and minimises risk of commiting passwords accidentally and might help prevent confusion over passwords

Reply:
Works on macOS too so I will change it.